decidim-core/app/commands/decidim/invite_user_to_private_participatory_space.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Decidim
+  # A command with the business logic to invite a user to
+  # a private participatory space.
+  class InviteUserToPrivateParticipatorySpace < Rectify::Command
```

Comment:
I would keep using InviteUserAgain and solve the problem there and get rid of this class

Reply:
Done!